applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -28,6 +28,21 @@ config BLE_LE_POWER_CONTROL_ENABLED
 	  dynamically and automatically during connection, which provides effective
 	  communication.
 
+config BT_DEVICE_NAME_ALT
```

Comment:
I get the idea.
If I am not mistaken, this solution supports two gateways only, and the user has to compile twice with different settings each time.

 Though, what about something like this: A random-static string is appended to the end of the device name in all cases. As the headset scans, it will get e.g.:
```
NRF5340_AUDIO_BROADCAST_0AF54E
NRF5340_AUDIO_BROADCAST_740C32
NRF5340_AUDIO_BROADCAST_BB8732

```
This means that the headset can toggle through N gateways, and there will be no need for two separate gateway compilations.

Reply:
The name is used to receive from a specific gateway and in this case, it also allows reception from two specific devices. In the headset we test against this name when scanning for gateways and hence these random names would need to be included within the headset so this compare could be performed. The reason for this approach is that there is no control layer informing which gateway and BIS to connect to.

Reply:
We discussed this, and decided to keep it as-is for now. May be improved in a separate PR